packages/orchestration/src/facade.js

```diff
@@ -78,14 +81,18 @@ export const makeOrchestrationFacade = ({
     const [wrappedCtx] = prepareEndowment(subZone, 'endowments', [hostCtx]);
     const hostFn = asyncFlow(subZone, 'asyncFlow', guestFn);
 
+    const zcfLtdContext = deepMapObject(wrappedCtx, val =>
+      val === zcf ? zcfLtd : val,
+    );
```

Comment:
In https://github.com/Agoric/agoric-sdk/issues/9773#issuecomment-2338946998 we captured the decision to not go the route of implicit mapping, instead requiring the contract to explicitly use the attenuated zcf instance